containers/datalab/content/GCSbackup.sh

```diff
@@ -123,7 +123,11 @@ fi
 # test and create bucket if necessary
 gsutil ls gs://${gcs_bucket} &>/dev/null || {
   echo "Bucket '${gcs_bucket}' was not found. Creating it.."
-  gsutil mb gs://"${gcs_bucket}"
+  gsutil mb gs://"${gcs_bucket}" || {
```

Comment:
I'd suggest a slightly different ordering of the logic where we only try to run `gsutil mb` once:

Basically, we'd look at the "gcs_bucket" (without any default), and "project_id" environment variables to figure out what the bucket name should be.

If "gcs_bucket" is defined, then that is the bucket name. Otherwise we run `gsutil ls gs://${project_id}.appspot.com` to see if that bucket exists, and if so set the bucket name to it. Finally, if neither of the two previous tests passed, then we set the bucket name to match the project ID.